internal/model/comments.go

```diff
@@ -0,0 +1,21 @@
+package model
+
+import (
+	"context"
+	"time"
+)
+
+type CommentsRepository interface {
+	Create(ctx context.Context, user_id int64, story_id int64, comment string) (Comment, error)
```

Comment:
Please use struct Comment as a param, instead of passing data comment one by one

```suggestion
	Create(ctx context.Context, data Comment) (Comment, error)
```